src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -784,6 +866,28 @@ void ov::npuw::LLMCompiledModel::convert_stateful_lora_to_stateless(std::shared_
     model->add_parameters(new_parameters);
 }
 
+void ov::npuw::LLMCompiledModel::gemma_transformations(const std::shared_ptr<ov::Model>& model) {
```

Comment:
Nitpick: might be name as `parameterize_sliding_mask(...)`

Reply:
There could be more gemma specific transformations, so i'd prefer to leave it as is